### DISCUSSION TITLE
Spotify not working

### DISCUSSION THREAD

**Discussion by ghost** [rern/rAudio#1300] [Bug Reports]
Hi,

I am trying to get Spotify working, however on login I notice the following log;

`Spotifyd.service- A spotify playing daemon
     Loaded: loaded (/etc/systemd/system/spotifyd.service; enabled; preset: disabled)
     Active: active (running) since Sat 2023-05-20 11:14:36 CEST; 10min ago
   Main PID: 3020 (spotifyd)
      Tasks: 2 (limit: 1060)
        CPU: 74ms
     CGroup: /system.slice/spotifyd.service
             └─3020 /usr/bin/spotifyd --no-daemon

May 20 11:14:36 rAudio systemd[1]: Started A spotify playing daemon.
May 20 11:14:38 rAudio spotifyd[3020]: Loading config from "/etc/spotifyd.conf"
May 20 11:14:38 rAudio spotifyd[3020]: No username specified. Checking username_cmd
May 20 11:14:38 rAudio spotifyd[3020]: No username_cmd specified
May 20 11:14:38 rAudio spotifyd[3020]: No password specified. Checking password_cmd
May 20 11:14:38 rAudio spotifyd[3020]: No password_cmd specified
May 20 11:14:38 rAudio spotifyd[3020]: No proxy specified
May 20 11:14:38 rAudio spotifyd[3020]: Using alsa volume controller.
May 20 11:14:38 rAudio spotifyd[3020]: no usable credentials found, enabling discovery
May 20 11:14:38 rAudio spotifyd[3020]: Failed to register IPv6 receiver: Os { code: 97, kind: Uncategorized, message: "Address family not supported by protocol" }`

I am using rAudio 20230310 on a Raspberry Pi 3B conencted through Wifi.
Any suggestions on how to resolve this?
thanks!

**Comment by McPaHa** [comment]
Hi!

Have same problem and I did (at least I think) like instructed.

**Comment by foberleitner** [comment]
Have the same problem - Spotify stopped working when I did a fresh setup from scratch (for some reason my device would refuse doing updates since a few months ago).
Did the setup again today with release [i20231022](https://github.com/rern/rAudio/releases/tag/i20231022).
Setup of Spotify according to instructions

-> spotifyd can not be discovered by Spotify app on my Android as well as on MacOS and Win 10.

What I also noticed: Airplay also stopped working

**Comment by Sagnasty** [comment]
Hello - see if #1061 helps your situation.

Also, I have not had any issues using AirPlay.

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+
